src/ontology/mondo-edit.obo

```diff
@@ -168766,18 +168766,21 @@ subset: rare
 synonym: "congenital deformity of the thumb and congenital alopecia" RELATED [GARD:0005199]
 synonym: "hypotrichosis associated with congenital hypoplasia of the thumb" RELATED [GARD:0005199]
 synonym: "sparse hair, short stature, hypoplastic thumbs, single upper central incisor and abnormal skin pigmentation" RELATED [GARD:0005199]
-synonym: "thumb deformity and alopecia" RELATED [OMIM:188150]
+synonym: "thumb deformity and alopecia" EXACT [OMIM:188150]
 synonym: "thumb deformity, alopecia, pigmentation anomaly" RELATED [GARD:0005199]
```

Comment:
is there a reason for this merge that the term name of the obsoleted term was not added as a synonym?